.dev/config/HEMCO_Config.rc

```diff
@@ -37,7 +37,7 @@ Separator:                   /
 Unit tolerance:              1
 Negative values:             0
 Only unitless scale factors: false
-Verbose:                     3
```

Comment:
Now that the model's configured properly, there's no longer any need to print such verbose output. So much is printed that an interactive debug session takes much longer printing to screen than running code!